environments/staging/aat.yml

```diff
@@ -24,6 +24,16 @@ A:
     record:
       - "20.49.129.95"
 
+  - name: "cft-mtls-api-mgmt-appgw"
+    ttl: 300
+    record:
+      - "20.26.101.168"
+
+  - name: "cft-api-mgmt-appgw"
```

Comment:
Drop this ? dont think its configured atm

Reply:
dropped